package.json

```diff
@@ -42,7 +42,7 @@
     "postinstall": "opencollective postinstall"
   },
   "dependencies": {
-    "marked": "^0.5.1",
+    "marked": "^0.6.2",
```

Comment:
might make more sense to go to 0.7.0 because I am seeing this:
```
───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ marked                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.7.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ docsify-cli [dev]                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ docsify-cli > docsify > marked                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1076  
```